examples/draft-0-10-0/color/color.html

```diff
@@ -26,8 +26,8 @@
       color may be active at any time.
     </div>
     <div id="target"></div>
-    <script src="../../../node_modules/react/dist/react.js"></script>
-    <script src="../../../node_modules/react-dom/dist/react-dom.js"></script>
+    <script src="../../../node_modules/react/umd/react.production.min.js"></script>
```

Comment:
can we update this PR to use the unminified dependencies ?

Reply:
yeah, i'll make that change here quick